src/search/operator_counting/operator_counting_heuristic.cc

```diff
@@ -24,10 +24,11 @@ OperatorCountingHeuristic::OperatorCountingHeuristic(const Options &opts)
         variables.push_back(lp::LPVariable(0, infinity, op_cost));
     }
     named_vector::NamedVector<lp::LPConstraint> constraints;
+    lp::LinearProgram lp(lp::LPObjectiveSense::MINIMIZE, move(variables), move(constraints), infinity);
```

Comment:
The constraints vector is created and directly moved. Can we use `{}` or `named_vector::NamedVector<lp::LPConstraint>()` instead of `move(constraints)`?